reproducer.yml

```diff
@@ -55,3 +55,10 @@
   roles:
     - role: ci_setup
     - role: reproducer
+
+  post_tasks:
+    - name: Allow traffic from OSP VMs to OSP API (needed for shiftstack)
+      ansible.builtin.command:
```

Comment:
you'll need a `  # noqa: command-instead-of-module` in order to pass ansible-lint.